packages/customWidgets/calendar-web/src/components/CalendarContainer.ts

```diff
@@ -116,8 +116,19 @@ export default class CalendarContainer extends Component<Container.CalendarConta
             return Promise.resolve(undefined);
         }
 
-        return new Promise(resolve => {
-            mxObject.fetch(attributePath, (attributeValue: any): void => resolve(attributeValue));
+        return new Promise((resolve, reject) => {
+            mxObject.fetch(
+                attributePath,
+                (attributeValue: any): void => {
+                    if (attributeValue instanceof Error) {
+                        reject(attributeValue);
+                    } else {
+                        resolve(attributeValue);
+                    }
+                },
+                // @ts-ignore
+                (error: Error): void => reject(error)
```

Comment:
Can we somehow avoid `@ts-ignore` since that alters the compiler?  Maybe something similar to the above callback with 1) typing or 2) the `instanceof` check? Does Typescript complain about just passing `reject` as is?

Reply:
I got rid of the ts-ignore with typing.

Reply:
`mendix.lib.MxObject` is a class. That's why I decided to go for the omit in the typing.